cmd/eval.go

```diff
@@ -86,5 +39,62 @@ func formatReplacements(replacementsInput map[string]string) map[string][]string
 // validateFile is a simple wrapper to ensure the input/output of valid YAML
 func validateFile(file []byte) error {
 	var holder interface{}
-	return yaml.Unmarshal(file, &holder)
+	err := yaml.Unmarshal(file, &holder)
+	if holder == nil {
+		return ErrEmptyFile
+	}
+	return err
```

Comment:
Thanks for updating - I think this is fine for now, but we should aim to clean this functionality up a bit while we iterate. See https://github.com/operator-framework/combo/pull/24/files#r749580994 for more information on one approach.

Reply:
Also, we should add a test case for this, but we can do that as a follow-up.